docs/nddata/subclassing.rst

```diff
@@ -80,7 +80,7 @@ super property set the attribute afterwards::
     ...             raise ValueError('uncertainty must have the same shape as the data.')
     ...         # Call the setter of the super class in case it might contain some
     ...         # important logic (only True for meta, unit and uncertainty)
-    ...         super(NDDataUncertaintyShapeChecker, self.__class__).uncertainty.fset(self, value)
+    ...         NDData.uncertainty.fset(self, value)
```

Comment:
Note that you can't easily use `super` when just overriding the setter of a property. This is probably better than using `self.__class__` as second argument... At least it's shorter and doesn't look like a bug waiting to happen.

Reply:
If @mwcraig approves this, I don't see a problem with the rest. Thanks!

Reply:
Yeah, I'm wondering if I shouldn't revert that one again. Hardcoding the superclass also has it's drawbacks because it breaks the MRO...

Reply:
This indeed feels very hacky. My tendency would be to leave it in place for this PR (which looks all OK apart from this), and open a separate issue about it (unless @mwcraig has an immediate answer).

Reply:
I don't have a better solution...except perhaps removing the example :wink:. (I know, not really an option) Opening a separate issue in a moment.